streams/tg.m3u

```diff
@@ -17,3 +17,9 @@ https://hls.newworldtv.com/nw-sport-2/video/live.m3u8
 https://pvqyb34odz24-hls-live.5centscdn.com/doc2/tv.stream/playlist.m3u8
 #EXTINF:-1 tvg-id="",NW Sport 3 (576p)
 https://hls.newworldtv.com/nw-sport-3/video/live.m3u8
+#EXTINF:-1 tvg-id="",NW Transat  (576p) [Not 24/7]
+https://hls.newworldtv.com/nw-transnat/video/live.m3u8
+#EXTINF:-1 tvg-id="",NW Economie (576p)
+https://hls.newworldtv.com/nw-economie/video/live.m3u8
```

Comment:
Offline.

<img width="416" alt="image" src="https://github.com/iptv-org/iptv/assets/7253922/da6661cc-1cf3-476d-b536-4effcf430433">

Reply:
Playlist gets in, but no live data found afterwards.